webpack/utils/getEntrypoints.js

```diff
@@ -2,9 +2,9 @@
 
 const path = require("path")
 const fs = require("fs")
-const { isDevelopment, basePath } = require("../../src/lib/environment")
```

Comment:
Can we delete this file? Or if its used elsewhere mind updating references to point at this new `env` implementation?

Reply:
I will double-check this. The goal here was to separate build code from production code. I find that having a clear separation between these layers makes it easier to work with and prevents changes to application logic from breaking the build infrastructure.

Reply:
👍 this environment file was pretty much only used for build stuff and was introduced during our last webpack refactor. Def double check, but i think it can be deleted.

Reply:
@icirellik Were you able to look at replacinig `environment`?

Reply:
It's used in several locations outside of Webpack. I'll go ahead and remove the development items from it.